docs/changes/0.19.0.rst

```diff
@@ -0,0 +1,74 @@
+Changelog for QCoDeS 0.19.0
+===========================
+
+The October 2020 release of QCoDeS.
+
+
+Breaking Changes:
+_________________
+
+There are no breaking changes in this release of QCoDeS.
+
+
+New:
+____
+-  Notebook: Example of how buffered sweep could be done on param with setpoints (#2263)
+
+
+Improved:
+_________
+- Station: Take verbose error message when validating station yaml against schema (#2282)
+- Legacy: ensure that datasets have unique names to avoid potential race condition (#2276) 
+- Add optional shape metadata to dataset (#2145)
+
+Improved drivers:
+_________________
+- AMI430: `has_current_rating` has been removed in the driver to work with the new firmware version (#2291)
+- Small typo fix in alazer drivers (#2296) 
+- Yokogawa GS200: `source_mode` bug fix (#2143)
```

Comment:
The driver for the alazar 9440 has also landed

Reply:
is this #2301?

Reply:
No https://github.com/QCoDeS/Qcodes/pull/2258

Reply:
added

Reply:
and #2301 is removed from changelog